README.md

```diff
@@ -6,32 +6,165 @@ Please edit this README.md file with your answers to the worksheet questions.
 
 ### a
 
+A | B | C | A AND B | NOT C | A AND B AND NOT C
+---|---|---|---|---|---
+0 | 0 | 0 | 0 | 1 | 0
+0 | 1 | 0 | 0 | 1 | 0
+0 | 0 | 1 | 0 | 0 | 0
+0 | 1 | 1 | 0 | 0 | 0
+1 | 0 | 0 | 0 | 1 | 0
+1 | 1 | 0 | 1 | 1 | 1
+1 | 0 | 1 | 0 | 0 | 0
+1 | 1 | 1 | 1 | 0 | 0
+
 ### b
 
+A | B | C | NOT C | B AND NOT C | NOT (B AND NOT C) | A AND NOT (B AND NOT C)
+---|---|---|---|---|---|---
+0 | 0 | 0 | 1 | 0 | 1 | 0
+0 | 1 | 0 | 1 | 1 | 0 | 0
+0 | 0 | 1 | 0 | 0 | 1 | 0
+0 | 1 | 1 | 0 | 0 | 1 | 0
+1 | 0 | 0 | 1 | 0 | 1 | 1
+1 | 1 | 0 | 1 | 1 | 0 | 0
+1 | 0 | 1 | 0 | 0 | 1 | 1
+1 | 1 | 1 | 0 | 0 | 1 | 1
+
 ### c
```

Comment:
Some mistakes here -- it looks like your `A OR NOT B` column is actually `A AND NOT B`